src/Action_RandomizeIons.cpp

```diff
@@ -1,7 +1,11 @@
 #include <cmath> // sqrt
-#include <cstdlib> // random, srandom
+#include <cstdlib> // srandom, random
```

Comment:
I wonder if it would be better to just use the `Random_Number` class (from `Random.h`)? The only reason `rand()` is used was to match ptraj results - not sure how critical `rand()` is to maintaining backwards compatibility since I don't think you could specify the seed in ptraj anyway. Would require some rewriting and testing so maybe it should be addressed in a new issue.